vendor/knative.dev/operator/pkg/apis/operator/v1beta1/knativeeventing_lifecycle.go

```diff
@@ -132,7 +133,7 @@ func (es *KnativeEventingStatus) GetVersion() string {
 
 // SetVersion sets the currently installed version of the component.
 func (es *KnativeEventingStatus) SetVersion(version string) {
-	es.Version = version
+	es.Version = os.Getenv("KNATIVE_EVENTING_VERSION")
 }
```

Comment:
This and the serving one, allows decoupling the directory name from the version in the status and since we only support one version per CSV